### PR TITLE
Header feature

### PR DESCRIPTION
**Pull request by HillaryOkello**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+text.md
```
Comment (wazih237) [inline .gitignore:1]: Here you had to put the text.md file inside the .gitignore file, and the "This file should be ignored by github" was to be in the text.md

File: style.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.head {
+    color: blue;
+}
```
Comment (wazih237) [inline style.css:3]: Indentation error. This shoul be spacing

Comment (HillaryOkello) [reply style.css:3]: let me fix that

File: .gitignore (hunk 1 above)
Comment (HillaryOkello) [reply .gitignore:1]: No, the gitignore is a file where you list down the unnecessary files that should be ignored by git

**Pull request merged**